lib/src/utils.dart

```diff
@@ -65,6 +66,9 @@ Future<ProcessResult> runProc(
   var items = await Future.wait(<Future<Object>>[
     process.exitCode,
     byteStreamSplit(process.stdout).forEach((outLine) {
+      if (deduplicate && stdoutLines.contains(outLine)) {
```

Comment:
Why is this necessary? it seems like something that should be fixed else where...

Reply:
I've filed an issue for dartanalyzer, but I also want us to unblock from waiting for it to happen.

Reply:
Perhaps add a todo here w/ a pointer to the issue?